statsmodels/stats/proportion.py

```diff
@@ -630,11 +632,12 @@ def binom_test_reject_interval(value, nobs, alpha=0.05, alternative='two-sided')
     else :
         x_upp = nobs
 
-    return x_low, x_upp
+    return int(x_low), int(x_upp)
```

Comment:
this breaks vectorization

But I don't remember how much that is supported here.
In general I was trying to get the hypothesis test to work for vectorized cases.
However, sample size computation are not vectorized in general cases (when we need rootfinding).
(maybe power computation itself works vectorized)